hw/bsp/apollo3_evb/src/hal_bsp.c

```diff
@@ -21,47 +21,10 @@
 #include "os/mynewt.h"
 
 #include <hal/hal_bsp.h>
-#include <bsp/bsp.h>
-#include <mcu/hal_apollo3.h>
-#include "am_mcu_apollo.h"
-
-#if MYNEWT_VAL(SPI_0_MASTER) || MYNEWT_VAL(SPI_1_MASTER) || MYNEWT_VAL(SPI_2_MASTER) || MYNEWT_VAL(SPI_3_MASTER) || MYNEWT_VAL(SPI_4_MASTER) || MYNEWT_VAL(SPI_5_MASTER)
 #include <hal/hal_spi.h>
-#endif
-
-#if MYNEWT_VAL(I2C_0) || MYNEWT_VAL(I2C_1) || MYNEWT_VAL(I2C_2) || MYNEWT_VAL(I2C_3) || MYNEWT_VAL(I2C_4) || MYNEWT_VAL(I2C_5)
 #include <hal/hal_i2c.h>
-#endif
-
-#if MYNEWT_VAL(UART_0) || MYNEWT_VAL(UART_1)
-#include "uart/uart.h"
-#include "uart_hal/uart_hal.h"
-#endif
-
-#if MYNEWT_VAL(ADC_0)
-#include <adc/adc.h>
-#include <adc_apollo3/adc_apollo3.h>
-#endif
-
-#if MYNEWT_VAL(UART_0)
-static struct uart_dev os_bsp_uart0;
-static const struct apollo3_uart_cfg os_bsp_uart0_cfg = {
-    .suc_pin_tx = MYNEWT_VAL(UART_0_PIN_TX),
-    .suc_pin_rx = MYNEWT_VAL(UART_0_PIN_RX),
-    .suc_pin_rts = MYNEWT_VAL(UART_0_PIN_RTS),
-    .suc_pin_cts = MYNEWT_VAL(UART_0_PIN_CTS),
-};
-#endif
-
-#if MYNEWT_VAL(UART_1)
-static struct uart_dev os_bsp_uart1;
-static const struct apollo3_uart_cfg os_bsp_uart1_cfg = {
-    .suc_pin_tx = MYNEWT_VAL(UART_1_PIN_TX),
-    .suc_pin_rx = MYNEWT_VAL(UART_1_PIN_RX),
-    .suc_pin_rts = MYNEWT_VAL(UART_1_PIN_RTS),
-    .suc_pin_cts = MYNEWT_VAL(UART_1_PIN_CTS),
-};
-#endif
+#include <mcu/hal_apollo3.h>
+#include "../include/bsp/bsp.h"
```

Comment:
You can simplify this to 
```suggestion
#include "bsp/bsp.h"
```